.travis.yml

```diff
@@ -37,4 +37,3 @@ addons:
     packages:
       - xvfb
       - g++-4.8
-      - libgnome-keyring-dev
```

Comment:
And I think this is the deps for keytar use not for muon which will be precompiled before browser-laptop can use it.